### ISSUE TITLE
Question about the DataPreprocessing when training ImageNet

### ISSUE THREAD

**Issue by RobertLuo1** [valeoai/Maskgit-pytorch#14]
Hi, Thanks for your wonderful work！
I notice that maskgit uses RandomResizedandCrop for data augmentation. But I find that in the code you adopt Cropping and flipping and comment out the Normalization (since I think VQGAN is trained with normalization). https://github.com/valeoai/Maskgit-pytorch/blob/b0b2b3cc11cffd0b159f22dc1c6e73a7e8b53db3/Trainer/trainer.py#L80 I am curious about the reason behind.

Thanks in advance!

**Comment by llvictorll** [comment]
Hi! Actually, I was not aware that RandomResizedandCrop is available directly from Pytorch... It might work, but be aware of the ratio aspect, otherwise the model will also generated "distorted" images.  
About the official MaskGIT, since they do not realize the training code, I can not help you, sorry.
Best, 
Victor

**Comment by RobertLuo1** [comment]
Yeah, but the inner function of RandomResizedandCrop is cropped first then Resized. I think it is not quite the same with the current operation.  I think the more feasible way is still using Resize and RandomCrop which is utilized in VQGAN Repo
